chapter2/tests/t3/chapter2-t3-velero-eks-diff.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Task 3 - find namespaces without the backup
+set -eo pipefail
+
+# Downloading YAML, extracting already saved namespaces into array.
+# Here we are using the from_yaml filter to convert a multi-line string into a YAML data structure,
+# after which we can use the dot notation to access the value of includedNamespaces.
+# See my explanation in the PR 
+savedns=($(curl -sSL https://gist.github.com/dmitry-mightydevops/016139747b6cefdc94160607f95ede74/raw/velero.yaml \
```

Comment:
Your variables names `savedns` and `allns` are not clear and not follow to variables naming guide, please add better names and format them properly